exporter.go

```diff
@@ -91,20 +119,35 @@ func main() {
 		},
 	}
 
-	if *nginxPlus {
-		client, err := plusclient.NewNginxClient(httpClient, *scrapeURI)
-		if err != nil {
-			log.Fatalf("Could not create Nginx Plus Client: %v", err)
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, syscall.SIGTERM)
+	go func() {
+		log.Printf("SIGTERM received: %v. Exiting...", <-signalChan)
+		os.Exit(0)
+	}()
+
+	for {
+		clientName := "Nginx Client"
+		var err error
+		var cl interface{}
+
+		if *nginxPlus {
+			clientName = "Nginx Plus Client"
```

Comment:
`clientName` is changed here all the time. I would just use the string literal in the function instead of declaring this variable and changing it
`handleRetryOrExit(*nginxRetries, *nginxRetryInterval, "Nginx Client", err)`  What do you think?

Reply:
Makes sense